packages/contentstack-import/src/lib/import/content-types.js

```diff
@@ -82,6 +83,10 @@ importContentTypes.prototype = {
     });
 
     return new Promise(function (resolve, reject) {
+      if (self.contentTypes === undefined || Object.keys(self.contentTypes).length === 0) {
```

Comment:
@abhinav-from-contentstack Can you reverify this condition once?

it checks that **if (some_variable === undefined || Object.keys(some_variable).length === 0)** right. Ultimately if the first condition fails it's going to jump into or condition **(Object.keys(some_variable).length === 0)** to validate right, in that case it's going to fail. Because **Object.keys(undefined).length** is not a valid code.

What if my **some_variable(asset/any)** is ideally undefined, All we know is variables can be undefined as well. So better handle it to prevent such an issue.